react/features/base/logging/middleware.js

```diff
@@ -121,6 +134,7 @@ function _libWillInit({ getState }, next, action) {
 function _setLoggingConfig({ getState }, next, action) {
     const result = next(action);
     const newValue = getState()['features/base/logging'].config;
+    const isDebugEnabled = getState()['features/base/config'].debug;
```

Comment:
would you mind using 'testing.testMode' instead of 'debug' ? I know torture was using 'debug' for some time already, but there's this dedicated 'testing' section in the config now.